### PR TITLE
Build Android app with Capacitor.js and Github Actions

### PR DESCRIPTION
**Pull request by bludnic**
Replace the old approach of building Android app with a third-party service like PWABuilder. Now, we can build it natively using Capacitor.js and Github Actions.

- [x] feat(package.json): add `android:preview` and `android:build` scripts
- [x] docs(README.md): add info about Android app
- [x] chore(Capacitor): auto build Android app using Github Actions
- [x] feat(Capacitor, Android): generate app icons using `@capacitor/assets`
- [x] ~~feat(Capacitor): use native Camera API~~ Set CAMERA permission to allow using Web Camera API

https://trello.com/c/ULCMY5Oo/402-migrate-from-pwabuilder-to-capacitorjs

### CODE REVIEW

**Comment by bludnic** [comment]
[Migrate from PWABuilder to Capacitor.js](https://trello.com/c/ULCMY5Oo/402-migrate-from-pwabuilder-to-capacitorjs)

**Comment by bludnic** [comment]
- Upgraded Capacitor.js `v5 → v6`
- Upgraded `@capacitor/assets` `v2 → v3` and regenerated assets
- Solved CAMERA permission issue when trying to scan QR codes in the native android app

File: scripts/capacitor/build-android.mjs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import dotenv from 'dotenv'
+import { $ } from 'execa'
+
+dotenv.config({
+  path: 'capacitor.env'
+})
+
+run()
+
+async function run() {
```
[the remark this answers is not in the document]
Comment (bludnic) [reply scripts/capacitor/build-android.mjs:10]: I followed the approach used in our other `.mjs` files, where each file includes a `run()` method. This ensures style consistency.

[the remark this answers is not in the document]
Comment (bludnic) [reply scripts/capacitor/build-android.mjs:10]: Added `void` to `run()` command https://github.com/Adamant-im/adamant-im/pull/515/commits/eaf8dede96b017c1310ccb556e934f97aeb6de8e

**Comment by adamant-al** [comment]
A step ahead ⚡️

**Pull request merged**